src/Services/OrderBookInitializer.cs

```diff
@@ -61,20 +62,21 @@ public async Task InitOrderBooks(bool clearExistingRecords = true)
                 }
                 else
                 {
-                    await _log.WriteWarningAsync("OrderBookInitializer", "InitOrderBooks", "", "No orderbooks on init");
+                    _log.Warning("No orderbooks on init");
                 }
 
                 return;
             }
 
             var exception = response.ErrorException ?? new Exception(response.Content);
-            await _log.WriteErrorAsync("OrderBookInitializer", "InitOrderBooks", "", exception);
+            _log.Error("InitOrderBooks", exception);
 
             throw exception;
         }
 
         private void ClearExistingRecords()
         {
+            return;
```

Comment:
Is this a mistake? If no then it's better to just delete this method.

Reply:
Thx, this is a mistake (were added for debug reasons).